osbs/utils.py

```diff
@@ -316,12 +318,14 @@ def make_name_from_git(repo, branch, limit=53, separator='-'):
 
     repo = git_repo_humanish_part_from_uri(repo)
     branch = branch or 'unknown'
+    hash_str = b64encode(sha256((repo + branch).encode('utf-8')).digest()).decode('ascii')[:hash_size]
```

Comment:
Can we split this line into smaller pieces so it's easier to see what's going on?

While doing that, let's make sure to avoiding having the base64-encoded value contain any characters OpenShift won't like: specify `altchars=b'__'`.